databases/models/Channels.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Channels extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Channels.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true
+        },
+        channel_id:{
+            type: DataTypes.UUID,
+            allowNull: false,
+        },
+        created_by: {
+            type: DataTypes.STRING
+        },
+        createdAt: {
+            type: DataTypes.DATE,
+            field: "created_at",
+        },
+        updatedAt: {
+            type: DataTypes.DATE,
+            field: "updated_at",
+        },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've manually defined `createdAt` and `updatedAt` fields. Sequelize automatically adds these fields when `timestamps: true` is set in the model options (line 42). You can remove these manual definitions unless you need specific customization.

```diff
-        createdAt: {
-            type: DataTypes.DATE,
-            field: "created_at",
-        },
-        updatedAt: {
-            type: DataTypes.DATE,
-            field: "updated_at",
-        },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->